### PR TITLE
luke

### PR DESCRIPTION
**Pull request by luke-dowling**

### CODE REVIEW

File: luke/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Declare a variable: 'It's hard to say who is right in this matter'.
+let argument = `It's hard to say who is right in this matter`;
+
+// 0.  Find the first occurrence of the letter "a" in a string, starting the search at position 17.
+console.log(argument.slice(17).indexOf(`a`));
```
Comment (ciamac-da) [inline luke/index.js]: The first occurrence of the letter "a" starting from 17 is not 22 , but 39 !
Here is the correct solution:
console.log(argument.indexOf("a", 17)); //39

**Review by ciamac-da — CHANGES_REQUESTED** [review]

File: luke/index.js (hunk 1 above)
Comment (luke-dowling) [reply luke/index.js]: added changes thanks ciamac :)

Comment (ciamac-da) [reply luke/index.js]: Thanks:) No prob:)

**Review by ciamac-da — APPROVED** [review]
👍🏻

**Pull request merged**